client/checkout/classic/upe-split.js

```diff
@@ -848,3 +838,36 @@ jQuery( function ( $ ) {
 		}
 	} );
 } );
+
+/**
+ * Checks if the customer is using a saved payment method.
+ *
+ * @param {string} paymentMethodType Stripe payment method type ID.
+ * @return {boolean} Boolean indicating whether or not a saved payment method is being used.
+ */
+export function isUsingSavedPaymentMethod( paymentMethodType ) {
+	const ccSavedPaymentMethodSelector =
+		'#wc-woocommerce_payments-payment-token-new';
+	const sepaPaymentMethodSelector =
+		'#wc-woocommerce_payments_sepa_debit-payment-token-new';
+	const tokenizedPaymentMethods = [ 'sepa_debit', 'card' ];
+
+	// The payment method used cannot be saved, which means that we are not using saved tokens for sure.
+	if ( ! tokenizedPaymentMethods.includes( paymentMethodType ) ) {
+		return false;
+	}
+
+	if ( 'sepa_debit' === paymentMethodType ) {
+		return (
+			null !== document.querySelector( sepaPaymentMethodSelector ) &&
+			! document.querySelector( sepaPaymentMethodSelector ).checked
+		);
+	}
+
+	if ( 'card' === paymentMethodType ) {
+		return (
+			null !== document.querySelector( ccSavedPaymentMethodSelector ) &&
+			! document.querySelector( ccSavedPaymentMethodSelector ).checked
+		);
+	}
```

Comment:
```suggestion
	const prefix = '#wc-woocommerce_payments-payment';
	const suffix = '-token-new';
	const savedPaymentSelector = 'card' === paymentMethodType ? prefix + suffix : prefix + paymentMethodType + suffix;

	return (
		null !== document.querySelector( savedPaymentSelector ) &&
		! document.querySelector( savedPaymentSelector ).checked
	);
```
This appears to be working fine, but I'd suggest that it's not tremendously future-proofed. Right now only the SEPA and card payment methods support saved payment methods through WCPay. However, Bancontact, iDEAL, and Sofort all also support recurring payments through Stripe and it's hopefully only a matter of time until we switch them on.

In a glowing future where all payment gateways are combined into the split UPE gateway, we would have to add new blocks for each of these payment methods, when enabling saved payments for these new methods. This will probably get tiresome to maintain at some point. 

If it were up to me, I'd probably rewrite this in a more dynamic fashion, using this suggestion as an example. That being said, this is working effectively right now and your more specific code is probably more efficient than my suggestion right now as well. Let me know your thoughts: I'm eager to hear them!

Reply:
A pretty fair suggestion it is! I've updated the implementation 57f19b6ec20d21398b775790360a919a146ce0cc